System-Automation-Scripts/Class_buddy.py

```diff
@@ -0,0 +1,86 @@
+from selenium import webdriver
+import schedule,time
+import keyboard
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.chrome.options import Options
+
+#joining Credentials
+username="email@gmail.com"    #enter your mail id
+password="This is my password"      #enter your password
+
+#giving camera and microphone access to the meet
+opt=Options()
+opt.add_argument("--disable-infobars")
+opt.add_argument("start-maximized")
+opt.add_argument("--disable-extensions")
+# Pass the argument 1 to allow and 2 to block
+opt.add_experimental_option("prefs", { \
+    "profile.default_content_setting_values.media_stream_mic": 1, 
+    "profile.default_content_setting_values.media_stream_camera": 1, 
+    "profile.default_content_setting_values.notifications": 1 
+      })
+browser =webdriver.Chrome(chrome_options=opt,executable_path="C:\\Users\\Yuvraj\\Downloads\\softwares\\Webdrivers\\chromedriver_win32\\chromedriver.exe") #Enter the path of your chrome driver

```

Comment:
Update the path to : <path_to_chromedriver.exe>

Also,
Thank you for your patience! I took a short break because of my exams.

Amazing job @Yuvraj-kadale !
Before I approve the above, could you make a little change in your PR?
Add screenshots of your working script inside the PR folder of the created branch & try to add points of installation in your README.md file (try adding one if you haven't, ignore if created!)

& you're good to go :rocket:

Reply:
@Neilblaze I have done the required changes in the PR, kindly review!!